src/morphology/german/generateParticipleForm.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Adds a prefix and a suffix to a stem in order to create a past participle form
+ *
+ * @param {string}  stemmedWord The stemmed word on which to
+ * @param {Object}  affixes     The suffix and prefix data.
+ *
+ * @returns {string} The participle form.
+ */
+const addParticipleAffixes = function( stemmedWord, affixes ) {
+	return affixes.prefix.concat( stemmedWord ).concat( affixes.suffix );
```

Comment:
[According to MDN, the `+` operator is several times faster than using `concat` to concatenate strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance). 
It is also more readable:
```js
return affixes.prefix + stemmedWord + affixes.suffix;
```
Can you check if this is done anywhere else in this PR as well?

Reply:
Nice find! I changed it everywhere in this PR and I'll make sure to use assignment operators instead of `concat` in the future.